src/sdk/sdk.ts

```diff
@@ -22,16 +22,21 @@ import {
 import {NonceManager} from '../nonce-manager/nonce-manager'
 import {OrderNonce} from '../nonce-manager/types'
 import {FusionOrder} from '../fusion-order'
+import {WebSocketApi} from '../api/ws'
 
 export class FusionSDK {
     public readonly api: FusionApi
 
+    public readonly ws: WebSocketApi
```

Comment:
📓 think it will be much more convenient to just have sepearte namesapce for all WebSocket methods

Reply:
I agree
Better to remove `ws` from FusionSDK, since only resolvers will use it right now
They will be able to import `ActiveOrdersWebSocketApi` from another space